src/linux/elf.rs

```diff
@@ -397,6 +353,72 @@ impl ElfInfo {
             stack,
         })
     }
+
+    pub fn merge(left: ElfInfo, right: ElfInfo) -> ElfInfo {
+        // Just to avoid to iterate on the bigger
+        let (mut left, mut right) = if left.symbols.len() > right.symbols.len() {
+            (left, right)
+        } else {
+            (right, left)
+        };
```

Comment:
I assume the output will be the same regardless of the order of the two symbol tables. It would be nice to add a test to ensure that's the case though maybe it would be best to do that with actual data instead. I was thinking that besides the unit-test we might want to setup a small test suite where we run this against a bunch of known debuginfos and match the output. Anyway this is not something to block on now.